### DISCUSSION TITLE
ERROR: matcher.locator(...).all is not a function

### DISCUSSION THREAD

**Discussion by duytran** [codeceptjs/CodeceptJS#4261] [General]
I meet **matcher.locator(...).all is not a function** error when play around with page http://uitestingplayground.com/

modules: 

* codeceptjs@3.5.15
* playwright@1.18

Step define:

```
I.amOnPage(' http://uitestingplayground.com/')
I.click('//*[@id="overview"]/div/div[1]/div[1]/h3/a')
```
Output
```
 matcher.locator(...).all is not a function
      at Playwright.findElements (node_modules\codeceptjs\lib\helper\Playwright.js:3460:55)
      at Playwright.findClickable (node_modules\codeceptjs\lib\helper\Playwright.js:3522:47)
      at Playwright.proceedClick (node_modules\codeceptjs\lib\helper\Playwright.js:3490:35)
```

**Comment by incoming-th** [comment]
@duytran I tried this on my side and no issue

![image](https://github.com/codeceptjs/CodeceptJS/assets/26375917/e7dcdc84-30a2-46a2-938c-1adecad8d34f)

Could be the space in the URL from your example or try to downgrade?

"codeceptjs": "^3.5.14",
 "playwright": "^1.42.1"

**Reply by duytran** [reply]
@incoming-th I am appreciate when you take time to answer to me.
The issue come from my playwright version. I updated it to latest and everything work fine.
